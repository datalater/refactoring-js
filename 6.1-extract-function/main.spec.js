import printOwing from "./main";
import invoice from "./fixtures/invoice.json";

describe("6.1 extract function", () => {
  it("snapshot", () => {
    const result = printOwing(invoice);

    expect(result).toMatchSnapshot();
  });
});
