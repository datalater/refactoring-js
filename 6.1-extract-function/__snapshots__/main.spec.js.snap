// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`6.1 extract function snapshot 1`] = `
"
*****************
**** 고객 채무 ****
*****************
고객명: Min
채무액: 600
마감일: 1970. 1. 31.
"
`;
